app/views/followups/_followup.html.erb

```diff
@@ -78,22 +78,43 @@
     <% status = @info_request.calculate_status %>
     <% if status == 'waiting_response_overdue' %>
       <p>
-        <%= _('The response to your request has been <strong>delayed</strong>. ' \
-              'You can say that, by law, the authority should normally have ' \
-              'responded <strong>promptly</strong> and by ' \
-              '<strong>{{date}}</strong>',
-              :date=>simple_date(@info_request.date_response_required_by)) %>
-
-        (<%= link_to _('details'),
-                     help_requesting_path(anchor: 'quickly_response') %>).
+        <% if @info_request.public_body.not_subject_to_law? %>
+          <%= _('The response to your request has been <strong>delayed' \
+                '</strong>. Although the authority has no legal obligation ' \
+                'to reply, they should normally have responded by ' \
+                '<strong>{{date}}</strong>',
+                :date=>simple_date(@info_request.date_response_required_by)) %>
+
+          (<%= link_to _('details'),
+                       "#{help_requesting_path}#authorities" %>).
```

Comment:
This re-implements the broken `link_to` fixed in 8c23cceef88bf49d71ef6f90a1d13f888a0f4cb0.